src/gui/mzroll/main.cpp

```diff
@@ -36,12 +37,27 @@ Database DB;
 void customMessageHandler(QtMsgType type, const QMessageLogContext &context, const QString &msg);
 plog::MyAppender<plog::TxtFormatter> myAppender; // Create our custom appender. 
 
+void initializeLogger()
+{
+    QDir dir;
+    QString path = QStandardPaths::writableLocation(QStandardPaths::GenericConfigLocation) + QDir::separator() + \
+                   qApp->organizationName() + QDir::separator() + qApp->applicationName() + QDir::separator() + "logs" \
+                   + QDir::separator() ;
+
+    // Logs  won't be saved anywhere if this mkpath returns False;
+    dir.mkpath(path);
```

Comment:
@rish9511 Shouldn't we at least inform the user? Maybe later, but I'd suggest you add a TODO here.